install/third-party/dapr/install.yml

```diff
@@ -0,0 +1,13 @@
+id: third-party-dapr
```

Comment:
Hi @harrykimpel we've been in the process of deprecating install plans in favor of data sources, and quickstarts now require a data source over an install plan.

More information here: https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#data-sources-1

Reply:
@MichelLosier I kept the install plan for now, let me know if you want me to remove it. I also added a data source.